pkg/utils/env.go

```diff
@@ -37,3 +39,8 @@ func NewEnvironment() Environment {
 func (osEnv OSEnvironment) GetEnv(propertyName string) string {
 	return os.Getenv(propertyName)
 }
+
+func (osEnv OSEnvironment) GetUsername() string {
+	name, _ := user.Current()
```

Comment:
Why ignore the error ? We need it passed back ? And the code should handle it if it can't be obtained.